core/shared/src/main/scala/zio/FiberRef.scala

```diff
@@ -467,54 +467,73 @@ object FiberRef {
         self =>
         type Patch = Patch0
 
-        def combine(first: Patch, second: Patch): Patch =
+        final def combine(first: Patch, second: Patch): Patch =
```

Comment:
What about using a final class here instead of an anonymous one? 🤔
Anonymous classes generates quite ugly stack traces
```scala
final class PatchFiber[Value] extends FiberRef[Value] {
  ...
}
```

Reply:
I thought about that too, I'll change it 👍